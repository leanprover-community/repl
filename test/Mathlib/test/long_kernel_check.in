{"cmd": "import Mathlib\nset_option maxHeartbeats 0\nset_option maxRecDepth 100000"}

{"cmd": "theorem amc12a_2003_p1 (u v : ℕ → ℕ) (h₀ : ∀ n, u n = 2 * n + 2) (h₁ : ∀ n, v n = 2 * n + 1) :\n    ((∑ k in Finset.range 2003, u k) - ∑ k in Finset.range 2003, v k) = 2003 := by sorry", "env": 0}

{"tactic": "(\nsimp only [h₀, h₁, Finset.sum_range_succ, Finset.sum_range_zero]; rfl)", "proofState": 0, "timeout": 2000}

