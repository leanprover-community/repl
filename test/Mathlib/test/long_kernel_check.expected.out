{"env": 0}

{"sorries":
 [{"proofState": 0,
   "pos": {"line": 2, "column": 83},
   "goal":
   "u v : ℕ → ℕ\nh₀ : ∀ (n : ℕ), u n = 2 * n + 2\nh₁ : ∀ (n : ℕ), v n = 2 * n + 1\n⊢ ∑ k ∈ Finset.range 2003, u k - ∑ k ∈ Finset.range 2003, v k = 2003",
   "endPos": {"line": 2, "column": 88}}],
 "messages":
 [{"severity": "info",
   "pos": {"line": 2, "column": 6},
   "endPos": {"line": 2, "column": 7},
   "data":
   "The '∑ x in s, f x' notation is deprecated: please use '∑ x ∈ s, f x' instead:\n∑ k ∈ Finset.range 2003, u k"},
  {"severity": "info",
   "pos": {"line": 2, "column": 39},
   "endPos": {"line": 2, "column": 40},
   "data":
   "The '∑ x in s, f x' notation is deprecated: please use '∑ x ∈ s, f x' instead:\n∑ k ∈ Finset.range 2003, v k"},
  {"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 22},
   "data": "declaration uses 'sorry'"}],
 "env": 1}

{"message": "Operation timed out"}

